tests/unit/lms/models/h_user_test.py

```diff
@@ -1,8 +1,8 @@
-from lms.models import HUser
+from tests import factories
 
 
 class TestHUser:
     def test_userid(self):
-        h_user = HUser("test_authority", "test_username")
+        h_user = factories.HUser(username="test_username", authority="test_authority")
```

Comment:
I think using the `kwargs` style is nicer for the reader.

Reply:
By the way, there is a way to get Factory Boy to pass arguments to the underlying model class as pargs (while still allowing the test code to pass arguments to the test factory as kwargs). You might have to use this if the underlying thing that your test factory calls has positional-only args: https://factoryboy.readthedocs.io/en/latest/reference.html#factory.FactoryOptions.inline_args